include/pocl_remote.h

```diff
@@ -32,4 +36,28 @@
 
 #define MAX_REMOTE_PARAM_LENGTH 256
 
+#if defined(ENABLE_REMOTE_DISCOVERY_ANDROID)
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+  /**
+   * Helper function used to dynamically add a remote server and its devices.
+   * It is currently only used by android where the network discovery is
+   * performed using android network libraries and the discovered servers are
+   * added using this function. The function is implemented in the remote
+   * driver in network_discovery.c.
+   */
+  void pocl_remote_discovery_add_server (const char *id,
```

Comment:
is this api intended for thirdparties to use as well? a description of the parameters would be useful.

Reply:
I'll add the description.

Could you elaborate on the 3rd parties?

Reply:
are only pocl developers going to use this method, or also other people?

Reply:
For now only pocl developers, for others to use it, it would have to be an extension.